src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +17,14 @@
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers
+                .stream()
+                .map(e -> e.split(","))
+                .flatMap(e -> Arrays.stream(e))
+                .map(e -> Integer.parseInt(e))
```

Comment:
```suggestion
        return numbers.stream()
		.flatMap(n -> Arrays.stream(n.split(",")))
                .map(e -> Integer.parseInt(e))
```

we can combine map and flatMap here